core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -1282,6 +1287,12 @@ private void printTableScanInfo(NodeRepresentation nodeOutput, TableScanNode nod
                             });
                 }
             }
+            PlanNodeStats nodeStats = stats.map(s -> s.get(node.getId())).orElse(null);
+            if (nodeStats != null && nodeStats.getPlanNodePhysicalInputDataSize().toBytes() > 0) {
+                nodeOutput.appendDetails(
```

Comment:
printTableScanInfo is called in `visitScanFilterAndProjectInfo` as well, so this might end up double printing `Physical Input` for scanfilterproject nodes.
You can make is this change in `visitTableScan` after the call to printTableScanInfo.
It'll also be useful to include Input positions for tablescan node as it's currently missing that.